golden_response/staging/get_landing_page_data/asm.json

```diff
@@ -611,7 +626,165 @@
       "RetailDrugDistribution_DrugDistribution_Oxycodone": {},
       "UnemploymentRate_Person": {},
       "UnemploymentRate_Person_Female": {},
-      "UnemploymentRate_Person_Male": {}
+      "UnemploymentRate_Person_Male": {},
+      "WorldBank_BM_TRF_PWKR_CD_DT": {},
```

Comment:
@shifucun any idea why this didn't have WDI data before?

Reply:
This cache flume should have use the updated chart config with these stat vars?